### PR TITLE
fixing some typos in the readme

### PR DESCRIPTION
**Pull request by zifn**
this is a hotfix

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +8,7 @@ Assignment: Project Family F, CS289A, UC Berkeley Fall 2020
...
+This codebase implements an analysis of COLd Target Recoil Ion Momentum Spectroscopy (COLTRIMS) scattering data by clustering measurements into potential molecular states. We use the cross-entropy loss function to as a measure of the goodness of fit. This is computed by using the clustering label generated from the clustering algorithm, and the probabilities generated after fitting each cluster to an angular probability distribution to compute the cross-entropy of the clustered results. The data analyzed in the final report was graciously provided by the [Atomic, Molecular, and Optical Sciences group](http://amo-csd.lbl.gov/home.php) at LBNL and has not yet been published. Thus it cannot be shared.
```
Comment (sajantanand) [inline README.md]: "We use the cross-entropy loss function to as a measure of the goodness of fit." is not correct.

**Review by sajantanand — COMMENTED** [review]
Just one grammatical edit.

**Review by sajantanand — APPROVED** [review]
Looks good to me.
